news.yaml

```diff
@@ -2498,7 +2498,7 @@ description: |
 ---
 title: |
   The Legal Framework for Reproducible Scientific Research: Licensing and Copyright
-link: http://doi.ieeecomputersociety.org/10.1109/MCSE.2009.19
+link: https://doi.ieeecomputersociety.org/10.1109/MCSE.2009.19
```

Comment:
Should probably use `doi.org` as well, but it seems that it points to a different source?

Reply:
Both that and https://www.computer.org/csdl/mags/cs/2009/01/mcs2009010035-abs.html are pay-walled, but https://web.stanford.edu/~vcs/papers/LFRSR12012008.pdf is OA :-)

Reply:
Should that be used instead? Or just the DOI?

Reply:
That question hasn't been answered since the inception of the DOI system itself :-D
It just an outsourced HTML redirect, so if the target server can keep it up, the DOI is not needed.